### DISCUSSION TITLE
Will not open .mp4 files from Matrice 300

### DISCUSSION THREAD

**Discussion by geonerdist** [vzhd1701/gridplayer#29] [Q&A] [answered]
Thank you so much for this program. I've been looking for a way to simultaneously play all 4 videos that our Matrice 300 records (standard, thermal, zoom, and wide angles) at the same time. However it will not play these video files but they play just fine in VLC. Any help would be appreciated.

**Comment by vzhd1701** [comment]
Could you please provide a sample file?

**Reply by geonerdist** [reply]
Thanks for the reminder, I was waiting for it to upload and got distracted.
https://1drv.ms/v/s!AglxpY8VKhXIg6NGTfLcrg06u05jzA?e=k8dQhr

**Comment by geonerdist** [comment, marked as answer]
@vzhd1701 Did that link work for you?

**Reply by vzhd1701** [reply]
Sorry for slow response. Your bug is fixed now, please check out the new version!

**Reply by geonerdist** [reply]
No worries, thanks so much for getting it fixed! It's working perfectly.
